src/content/docs/alerts-applied-intelligence/overview.mdx

```diff
@@ -24,85 +24,67 @@ redirects:
-</LandingPageTile>
-
-{' '}
-<LandingPageTile
-  title="Create notification channels."
-  href="/docs/alerts/new-relic-alerts/managing-notification-channels/notification-channels-controlling-where-send-alerts"
-  icon="fe-message-circle"
->
-  Specify how to get notified when critical violations occur, so you can quickly
-  fix the issue.
-</LandingPageTile>
-
-  <LandingPageTile
-    title="Learn about AI."
-    href="/docs/alerts-applied-intelligence/new-relic-alerts/get-started/introduction-applied-intelligence"
-    icon="fe-alert-triangle"
-  >
-
-    Take advantage of automatically enabled [Proactive Detection](/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-new-relic-ai) to be warned about trouble brewing in your APM-monitored apps. Group similar incidents with [Incident intelligence](/docs/alerts-applied-intelligence/incident-intelligence/get-started-incident-intelligence). [Incident workflows](/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/enhance-notifications-incident-workflows) enrich your notifications with New Relic data.
-
-  </LandingPageTile>
-
-  <LandingPageTile
-    title="Resolve an issue."
-    href="/docs/alerts/new-relic-alerts/reviewing-alert-incidents"
-    icon="fe-tool"
-  >
-     [Acknowledge an incident](/docs/alerts/new-relic-alerts/reviewing-alert-incidents/acknowledge-alert-incidents) after it occurs, or [close a violation manually](/docs/alerts/new-relic-alerts/reviewing-alert-incidents/close-violations-manually) as needed.
-  </LandingPageTile>
-</LandingPageTileGrid>
+
+In the New Relic UI, Alerts & AI is divided into several sections, each unlocking different features. Check out the section explanations below to learn how these features can meet your needs.
+
+## **Analyze** your issues, incidents, and anomalies [#analyze]
+
+Use the analyze features to scan for critical problems affecting your systems, via alerts violations, issues, incidents, and anomalies.
+
+* **[Overview](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page)**: Find your most recent open, closed, and muted violations.
+* **Issues and activity**: Review your most recent [issues](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#issues), [incidents](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#incidents), and [anomalies](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#anomalies).
+
+## **Detect** problems in your system [#detect]
+
+Use the detect features to define the conditions in your system you want to be notified about, as well as to configure how anomalies are detected.
+
+* **[Alert conditions (policies)](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/)**: Create new policies. Also, lists the policies where your conditions are created and organized. 
```

Comment:
@paperclypse, is there a way to succinctly clarify the relationship between policies and conditions here? Are policies collections of conditions?

Reply:
Good call!